acceptance/tests/apply/classes/should_allow_param_undef_override.rb

```diff
@@ -1,5 +1,10 @@
 test_name "should allow overriding a parameter to undef in inheritence"
 
+tag 'audit:high',
+    'audit:unit',  # This should be covered at the unit layer.
+    'audit:delete'
```

Comment:
is it covered at unit? if so, can this be deleted in this PR?

spurious newline